<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedMenu</class>
 <widget class="QStackedWidget" name="StackedMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background-color: rgba(90,90,90,250); border:none; }

QLabel#label_title {font: bold 34pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}

QPushButton#btn_exit { background-color: rgba(250,100,100,250); font: bold 18pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}
QPushButton#btn_exit:pressed { background-color: rgba(100,100,250,250); font: bold 18pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}

QPushButton#btn_saveexit { background-color: rgba(100,255,100,250); font: bold 18pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}
QPushButton#btn_saveexit:pressed { background-color: rgba(100,100,250,250); font: bold 18pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}

QPushButton#btn_back { border-image: url(:/ui/back.png); font: bold 12pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255); }
QPushButton#btn_back:pressed { border-image: url(:/ui/backlight.png); }

QWidget#line_2 { background-color: rgba(44,44,44,250); border:none; }

QLabel#label_video, QLabel#label_audio {font: bold 22pt  &quot;Walkway Expand UltraBold&quot;; color:rgb(255,255,255);}




</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page_menu">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="ItemScrollArea" name="scroll_menu">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="ItemScrollContent" name="scroll_menu_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>450</width>
         <height>450</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="line" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_saveexit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>Save and Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_exit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_para">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(50, 50, 50, 255), stop:0.505682 rgba(120, 120, 120, 255), stop:0.982955 rgba(90, 90, 90, 255)); border:none; }

QPushButton#btn_back_2 { background-color: rgba(82, 64, 209, 0); border:none; }
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_back">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>52</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string> Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_back_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>52</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="line_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked_menu_paras">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_menu_time">
       <property name="styleSheet">
        <string notr="true">/*ItemScrollArea {border-image: url(:/ui/timebg.png);}*/</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_year">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_year">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>120</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_month">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_month">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>80</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_day">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_day">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>80</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_hour">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_hour">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>80</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_minute">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_minute">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>80</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_menu_volume">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="wid_video" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>214</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>214</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_video">
               <property name="text">
                <string>Video</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ItemScrollArea" name="scroll_volume_video">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>180</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>180</height>
                </size>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="ItemScrollContent" name="scrollArea_volume_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>120</width>
                  <height>180</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wid_audio" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>214</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>214</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_audio">
               <property name="text">
                <string>Audio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ItemScrollArea" name="scroll_volume_audio">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>180</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>180</height>
                </size>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="ItemScrollContent" name="scrollArea_volume">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>120</width>
                  <height>180</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_menu_backlight">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="ItemScrollArea" name="scroll_backlight">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_backlight">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>150</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_menu_theme"/>
      <widget class="QWidget" name="page_menu_protocol">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_protocol_type">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_protocol_type">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>150</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_protocol_buad">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_protocol_buad">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>120</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="ItemScrollArea" name="scroll_protocol_id">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>180</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="ItemScrollContent" name="scrollArea_protocol_id">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>120</width>
               <height>180</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemScrollArea</class>
   <extends>QScrollArea</extends>
   <header>ilistview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemScrollContent</class>
   <extends>QWidget</extends>
   <header>ilistview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
